python/private/hermetic_runtime_repo_setup.bzl

```diff
@@ -184,6 +184,10 @@ def define_hermetic_runtime_toolchain_impl(
                 "lib/libpython{major}.{minor}.so.1.0".format(**version_dict),
             ],
             "@platforms//os:macos": ["lib/libpython{major}.{minor}.dylib".format(**version_dict)],
+            "@platforms//os:qnx": [
```

Comment:
Could you please refactor the code so that this is the default? It seems that `linux` and `qnx` have the same code here, so in theory we could just default to that. I wonder if that would incidentally also help FreeBSD, which we don't support, but you never know what PR will come in the future.

@rickeylev, what would you think about this?